README.md

```diff
@@ -1,15 +1,65 @@
-# ROS2 Wrapper for Intel&reg; RealSense&trade; Devices
-These are packages for using Intel RealSense cameras (D400, L500 and SR300 cameras) with ROS2.
-
-This version supports ROS2 Dashing, Eloquent, Foxy, Galactic and Rolling.
-
-LibRealSense supported version: v2.51.1 (see [realsense2_camera release notes](https://github.com/IntelRealSense/realsense-ros/releases))
-
-
-## For LibRS ROS1 Wrapper please refer to [ROS1-legacy branch](https://github.com/IntelRealSense/realsense-ros/tree/ros1-legacy)
-
-## Please notice: if you are moving from RealSense [ROS2-legacy branch](https://github.com/IntelRealSense/realsense-ros/tree/ros2-legacy) to ROS2-development:
-- **Changed Parameters**:
+<h1 align="center">
+   <img src="https://www.intelrealsense.com/wp-content/uploads/2020/09/intel-realsense-logo-360px.png" alt="Intel® RealSense™" title="Intel® RealSense™" />
+</h1>
+
+<p align="center">
+  ROS2 packages for using Intel RealSense D400 cameras.<br>
+  <a href="https://github.com/IntelRealSense/realsense-ros/releases">Latest release notes</a>
+</p>
+
+<hr>
+
+<div style="text-align:center">
+
+[![rolling][rolling-badge]][rolling]
+[![humble][humble-badge]][humble]
+[![foxy][foxy-badge]][foxy]
+[![galactic][galactic-badge]][galactic]
+[![eloquent][eloquent-badge]][eloquent]
+[![dashing][dashing-badge]][dashing]
+[![ubuntu22][ubuntu22-badge]][ubuntu22]
+[![ubuntu20][ubuntu20-badge]][ubuntu20]
+[![ubuntu18][ubuntu18-badge]][ubuntu18]
+
+![GitHubWorkflowStatus](https://img.shields.io/github/actions/workflow/status/IntelRealSense/realsense-ros/main.yml?logo=github&style=flat-square)
+![GitHubcontributors](https://img.shields.io/github/contributors/IntelRealSense/realsense-ros?style=flat-square)
+![License](https://img.shields.io/github/license/IntelRealSense/realsense-ros?style=flat-square)
+
+</div>
+
+
+## Table of contents
+  * [ROS1 and ROS2 legacy](#legacy)
+  * [Installation](#installation)
+  * [Usage](#usage)
+     * [Starting the camera node](#start-camera-node)
+     * [Parameters](#parameters)
+     * [Topics](#topics)
+     * [Metadata Topic](#metadata)
+     * [Post-Processing Filters](#filters)
+     * [Available Services](#services)
+     * [Efficient intra-process communication](#intra-process)

```

Comment:
We can here add some coordination system section/point to doc.
Better to explain in the doc Vision sensors + IMU axis

Reply:
done